index.html

```diff
@@ -30,18 +30,29 @@
             alt="logo"
           />
         </div>
-        <div class="box-hamburger">
-          <i class="fa-solid fa-bars hamburger"></i>
-        </div>
+
         <ul class="second_nav_deskt">
           <li><a href="about.html">About</a></li>
-          <li><a href="#">Program</a></li>
-          <li><a href="#">Join</a></li>
-          <li><a href="#">Sponsor</a></li>
+          <li><a href="#main-program">Program</a></li>
+          <li><a href="#speakers">Speakers</a></li>
+          <li><a href="#partners">Sponsor</a></li>
           <li><a href="#">News</a></li>
           <li><a href="#">DevOps Campaign</a></li>
         </ul>
       </nav>
+      <nav class="mobile-nav">
+        <div class="hamburger">
+          <img id="hamburger" src="./assets/images/ham.svg" alt="" />
```

Comment:
- [ ] Well done on including the `alt` attribute for all your images! **It's a crucial step in improving the accessibility of your webpage.** I would like to kindly suggest adding the `alt` attribute to **the remaining image in this section as well.** By doing so, **you'll further enhance the accessibility and inclusivity of your webpage,** making it easier for users with visual impairments to understand the content. 🌟